.devcontainer/download_libtorch.sh

```
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -ev
+
+SCRIPT_PATH=$(dirname $(realpath -s $0))
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)